File: repos_estudiantes_TARQUI_primera_entrega/Quispe_Mollisaca_Jimmy_Brandon/Introduccion Programacion Orientada a Objetos/Ejer2/Coche.java
package IntroPOO;

public class Coche {
	
	private 
	String marca;
	String modelo;
	int velocidad;
	
	public Coche(String marca, String modelo, int velocidad) {
		super();
		this.marca = marca;
		this.modelo = modelo;
		this.velocidad = velocidad;
	}
	
	
	public  void acelerar(int a) {
		this.velocidad += a;
	}
	
	public void frenar(int f) {
		if(velocidad >= f) {
			this.velocidad -= f;	
		}
		else {
			this.velocidad=0;
		}
	}
	public void mostrarVelocidad() {
        System.out.println("Velocidad del coche (" + marca + " " + modelo + "): " + velocidad + " km/h");
    }
	
	public static void main(String[] args) {
		Coche coche1 = new Coche("Toyota", "Corolla", 30);
        Coche coche2 = new Coche("Ford", "Focus", 50);

        coche1.acelerar(10);
        coche2.acelerar(10);

        coche1.frenar(5);
        coche2.frenar(5);
        
        System.out.println("--------Coches--------");

        coche1.mostrarVelocidad();
        coche2.mostrarVelocidad();

	}

}

File: repos_estudiantes_TARQUI_primera_entrega/Quispe_Mollisaca_Jimmy_Brandon/Introduccion Programacion Orientada a Objetos/Ejer2/Coche.py
class Coche:
    def __init__(self, marca, modelo, velocidad):
        self.marca = marca
        self.modelo = modelo
        self.velocidad = velocidad

    def acelerar(self, a):
        self.velocidad += a

    def frenar(self, f):
        if self.velocidad >= f:
            self.velocidad -= f
        else:
            self.velocidad = 0

    def mostrar_velocidad(self):
        print(f"Velocidad del coche ({self.marca} {self.modelo}): {self.velocidad} km/h")


coche1 = Coche("Toyota", "Corolla", 30)
coche2 = Coche("Ford", "Focus", 50)

coche1.acelerar(10)
coche2.acelerar(10)

coche1.frenar(5)
coche2.frenar(5)

print("--------Coches--------")
coche1.mostrar_velocidad()
coche2.mostrar_velocidad()